packages/twenty-website/src/content/user-guide/getting-started/book-your-onboarding-call.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Book your Onboarding Call
+info: Get access to the team's calendar to get some help
+icon: IconNote
+image: /images/user-guide/import-export-data/cloud.png
+sectionInfo: Discover Twenty, an open-source CRM.
+---
+
+## Book your onboarding call
+Whether you need help getting started or evaluating if Twenty is the CRM you need, we'll be happy to assist.  
+**Just book a call [here](https://cal.com/stephaniejoly/15min).** 
```

Comment:
Let's create a dedicated team page even if it's just you behind this - it will look more professional and means we don't have to update this as the team grows in the future